generic_tracers/cobalt_glbl.F90

```diff
@@ -0,0 +1,1531 @@
+!> COBALT_glbl module consists of core parameters and vars 
+!! to be used by generic COBALT related modules
+!<----------------------------------------------------------------
+module cobalt_glbl
```

Comment:
Following our conversation yesterday, do we want to call this module "cobalt_types"?

Reply:
rename `cobalt_glbl` to `cobalt_types`